implementations/rust/ockam/ockam_command/src/secure_channel/listener/list.rs

```diff
@@ -30,12 +30,12 @@ const AFTER_LONG_HELP: &str = include_str!("./static/list/after_long_help.txt");
 pub struct ListCommand {
     /// Node of which secure listeners shall be listed
     #[command(flatten)]
-    node_opts: NodeOpts,
+    node_opts: SecureChannelListenerNodeOpts,
```

Comment:
Hey @leafoliage, thanks for taking the time looking into this!

This change is actually not needed anymore after https://github.com/build-trust/ockam/pull/5034 was landed. The `NodeOpts` struct argument was renamed to `--at`.

One thing you could do though that would be a great improvement, would be to remove the `SecureChannelListenerNodeOpts` struct and replace its usages by `NodeOpts`.

Reply:
ok! will do.